### DISCUSSION TITLE
Can I delete log, trace, crash reports files ?

### DISCUSSION THREAD

**Discussion by sammbusy** [apohl79/audiogridder#313] [Q&A] [answered]
Hello guys, simple question, can I delete the .log, .trace files ?  Also the .dmp files on my Windows virtual machine ? 

They take quite a lot of space for examples 300MB for .dmp files, 56MB for .trace files, etc...

Thank you

**Comment by apohl79** [comment, marked as answer]
Yes you can delete them all. Since beta9 or 10 AG is just keeping the last 5 logs and traces and last 3 dmp files.
